src/sentry/deletions/defaults/group.py

```diff
@@ -55,15 +75,26 @@
     models.GroupOwner,
     models.GroupEmailThread,
     models.GroupSubscription,
-    models.GroupHistory,
```

Comment:
It's repeated on the list:
https://github.com/getsentry/sentry/blob/89c7bed0b62de891c41d4b0c9b491855d3c23b4e/src/sentry/deletions/defaults/group.py#L42